### ISSUE TITLE
Feature Request: Clear Video Input

### ISSUE THREAD

**Issue by Brotuulaan** [bitfocus/companion-module-renewedvision-propresenter#138]
It would be nice to be able to clear the video input via Companion just like the rest of the clear buttons available in Pro7.

**Comment by greyshirtguy** [comment]
This is not supported in the reverse engineered remote protocol that the module currently usees.
It is added to the new Pro7 API.... but I'll prob make a new module to support that new API in Pro7 (for many reasons).

In the meantime, if you are running the later versions of Pro7 that support the new API - it's easy to add a button that does this using the Companion Generic HTTP module and calling the new Pro7 API directly. Let me know in a reply if you want instructions/demot to set this up.

**Comment by Brotuulaan** [comment]
Yes, a link to a video or article on how to do this would be great! Is that something they have available on their site somewhere? Is it ProPresenter Control that you're referring to?
________________________________

**Comment by greyshirtguy** [comment]
There is no video or article...so I just made a demo video (totally unscripted - but should still be useful).
https://www.dropbox.com/s/ua9oedzugm4u9as/Demo%20using%20generic%20http%20module%20to%20call%20pro7%20API.mov?dl=0
